scripts/core/ui/ui.js

```diff
@@ -1122,23 +1125,77 @@ function splitterWidget(superdesk, superdeskFlags, $timeout) {
                     }
 
                     authoring.width(authoringWidth / container.width() * 100 + '%');
+
+                    header.width(stage.outerWidth());
                 },
                 stop: function (e, ui) {
                     var container = ui.element.parent();
 
+                    var stage = ui.element.find('.stage.swimlane');
+                    var header = stage.find('.column-header.swimlane');
+
                     superdesk.monitoringWidth = workspace.outerWidth() / container.width() * 100 + '%';
                     superdesk.authoringWidth = authoring.outerWidth() / container.width() * 100 + '%';
 
+                    superdesk.headerWidth = superdesk.stageWidth = stage.outerWidth();
+
                     ui.element.css({
                         width: superdesk.monitoringWidth
                     });
+
+                    header.css({
+                        width: superdesk.headerWidth
+                    });
                 }
             });
         }
     };
 }
 
 /*
+ * Header Resize directive for swimlane view, applies the same width of
+ * stage column to its fixed positioned header in order to resize it.
+ * sd-header-resize used in monitoring-view.html with respective sd-monitoring-group
+ *
+ */
+headerResizeDirective.$inject = ['$rootScope', '$timeout', '$window', 'workspaces'];
+function headerResizeDirective($rootScope, $timeout, $window, workspaces) {
+    return {
+        link: function(scope, element) {
+            let window = angular.element($window);
+            let resize = _.debounce(calcSize, 250);
+            window.on('resize', resize);
+
+            function calcSize() {
```

Comment:
not sure if this can be solved with css, @pavlovicnemanja @Aca-jov any idea?
there is fixed position header that should fit in parent, whose width might change (based on number of columns).